diff --git a/src/ptests/CMakeLists.txt b/src/ptests/CMakeLists.txt
index d675c34..530c27a 100644
--- a/src/ptests/CMakeLists.txt
+++ b/src/ptests/CMakeLists.txt
@@ -77,42 +77,42 @@ endif (CGNS_ENABLE_FORTRAN AND HAVE_FORTRAN_2003)
 
 # Add the tests so that cmake can find them
 if (CGNS_ENABLE_TESTS)
-  add_test(pcgns_ctest ${MPIEXEC} ${MPIEXEC_NUMPROC_FLAG} "2"
+  add_test(pcgns_ctest ${MPIEXEC} -x LD_LIBRARY_PATH ${MPIEXEC_NUMPROC_FLAG} "2"
       ${MPIEXEC_PREFLAGS} ./pcgns_ctest ${MPIEXEC_POSTFLAGS})
-  add_test(cexample ${MPIEXEC} ${MPIEXEC_NUMPROC_FLAG} "2"
+  add_test(cexample ${MPIEXEC} -x LD_LIBRARY_PATH ${MPIEXEC_NUMPROC_FLAG} "2"
       ${MPIEXEC_PREFLAGS} ./cexample ${MPIEXEC_POSTFLAGS})
-  add_test(benchmark ${MPIEXEC} ${MPIEXEC_NUMPROC_FLAG} "2"
+  add_test(benchmark ${MPIEXEC} -x LD_LIBRARY_PATH ${MPIEXEC_NUMPROC_FLAG} "2"
       ${MPIEXEC_PREFLAGS} ./benchmark ${MPIEXEC_POSTFLAGS})
-  add_test(benchmark_hdf5 ${MPIEXEC} ${MPIEXEC_NUMPROC_FLAG} "2"
+  add_test(benchmark_hdf5 ${MPIEXEC} -x LD_LIBRARY_PATH ${MPIEXEC_NUMPROC_FLAG} "2"
       ${MPIEXEC_PREFLAGS} ./benchmark_hdf5 ${MPIEXEC_POSTFLAGS})
-  add_test(Metadata_Stressing ${MPIEXEC} ${MPIEXEC_NUMPROC_FLAG} "2"
+  add_test(Metadata_Stressing ${MPIEXEC} -x LD_LIBRARY_PATH ${MPIEXEC_NUMPROC_FLAG} "2"
       ${MPIEXEC_PREFLAGS} ./Metadata_Stressing ${MPIEXEC_POSTFLAGS})
-  add_test(open_close ${MPIEXEC} ${MPIEXEC_NUMPROC_FLAG} "2"
+  add_test(open_close ${MPIEXEC} -x LD_LIBRARY_PATH ${MPIEXEC_NUMPROC_FLAG} "2"
       ${MPIEXEC_PREFLAGS} ./open_close ${MPIEXEC_POSTFLAGS})
-  add_test(comm_test ${MPIEXEC} ${MPIEXEC_NUMPROC_FLAG} "2"
+  add_test(comm_test ${MPIEXEC} -x LD_LIBRARY_PATH ${MPIEXEC_NUMPROC_FLAG} "2"
       ${MPIEXEC_PREFLAGS} ./comm_test ${MPIEXEC_POSTFLAGS})
-  add_test(test_base ${MPIEXEC} ${MPIEXEC_NUMPROC_FLAG} "2"
+  add_test(test_base ${MPIEXEC} -x LD_LIBRARY_PATH ${MPIEXEC_NUMPROC_FLAG} "2"
       ${MPIEXEC_PREFLAGS} ./test_base ${MPIEXEC_POSTFLAGS})
-  add_test(test_unstructured ${MPIEXEC} ${MPIEXEC_NUMPROC_FLAG} "2"
+  add_test(test_unstructured ${MPIEXEC} -x LD_LIBRARY_PATH ${MPIEXEC_NUMPROC_FLAG} "2"
       ${MPIEXEC_PREFLAGS} ./test_unstructured ${MPIEXEC_POSTFLAGS})
-  add_test(test_unstruc_quad ${MPIEXEC} ${MPIEXEC_NUMPROC_FLAG} "2"
+  add_test(test_unstruc_quad ${MPIEXEC} -x LD_LIBRARY_PATH ${MPIEXEC_NUMPROC_FLAG} "2"
       ${MPIEXEC_PREFLAGS} ./test_unstruc_quad ${MPIEXEC_POSTFLAGS})
-  add_test(test_zone ${MPIEXEC} ${MPIEXEC_NUMPROC_FLAG} "2"
+  add_test(test_zone ${MPIEXEC} -x LD_LIBRARY_PATH ${MPIEXEC_NUMPROC_FLAG} "2"
       ${MPIEXEC_PREFLAGS} ./test_zone ${MPIEXEC_POSTFLAGS})
-  add_test(test_general_readwrite ${MPIEXEC} ${MPIEXEC_NUMPROC_FLAG} "2"
+  add_test(test_general_readwrite ${MPIEXEC} -x LD_LIBRARY_PATH ${MPIEXEC_NUMPROC_FLAG} "2"
       ${MPIEXEC_PREFLAGS} ./test_general_readwrite ${MPIEXEC_POSTFLAGS})
-  add_test(test_poly_unstructured ${MPIEXEC} ${MPIEXEC_NUMPROC_FLAG} "2"
+  add_test(test_poly_unstructured ${MPIEXEC} -x LD_LIBRARY_PATH ${MPIEXEC_NUMPROC_FLAG} "2"
       ${MPIEXEC_PREFLAGS} ./test_poly_unstructured ${MPIEXEC_POSTFLAGS})
-  add_test(thesis_benchmark ${MPIEXEC} ${MPIEXEC_NUMPROC_FLAG} "2"
+  add_test(thesis_benchmark ${MPIEXEC} -x LD_LIBRARY_PATH ${MPIEXEC_NUMPROC_FLAG} "2"
       ${MPIEXEC_PREFLAGS} ./thesis_benchmark ${MPIEXEC_POSTFLAGS})
   if (CGNS_ENABLE_FORTRAN  AND HAVE_FORTRAN_2003)
-    add_test(pcgns_ftest ${MPIEXEC} ${MPIEXEC_NUMPROC_FLAG} "2"
+    add_test(pcgns_ftest ${MPIEXEC} -x LD_LIBRARY_PATH ${MPIEXEC_NUMPROC_FLAG} "2"
         ${MPIEXEC_PREFLAGS} ./pcgns_ftest ${MPIEXEC_POSTFLAGS})
-    add_test(fexample ${MPIEXEC} ${MPIEXEC_NUMPROC_FLAG} "2"
+    add_test(fexample ${MPIEXEC} -x LD_LIBRARY_PATH ${MPIEXEC_NUMPROC_FLAG} "2"
         ${MPIEXEC_PREFLAGS} ./fexample ${MPIEXEC_POSTFLAGS})
-    add_test(benchmark_hdf5_f90 ${MPIEXEC} ${MPIEXEC_NUMPROC_FLAG} "2"
+    add_test(benchmark_hdf5_f90 ${MPIEXEC} -x LD_LIBRARY_PATH ${MPIEXEC_NUMPROC_FLAG} "2"
         ${MPIEXEC_PREFLAGS} ./benchmark_hdf5_f90 ${MPIEXEC_POSTFLAGS})
-    add_test(test_mixed_par_ser ${MPIEXEC} ${MPIEXEC_NUMPROC_FLAG} "2"
+    add_test(test_mixed_par_ser ${MPIEXEC} -x LD_LIBRARY_PATH ${MPIEXEC_NUMPROC_FLAG} "2"
         ${MPIEXEC_PREFLAGS} ./test_mixed_par_ser ${MPIEXEC_POSTFLAGS})
   endif (CGNS_ENABLE_FORTRAN  AND HAVE_FORTRAN_2003)
 endif (CGNS_ENABLE_TESTS)
